### ISSUE TITLE
Can't debug containerized Python app in WSL without DockerDesktop

### ISSUE THREAD

**Issue by Wadzio** [microsoft/vscode-docker#3525]
OS: Windows 10 (21H2 build: 19044.1706)
VSCode: 1.67.1
VSCode-docker: 1.22.0
WSL2 with distro: Ubuntu 20.4, kernel 5.10.16
Docker in Ubuntu distro: 20.10.12
**_No DockerDesktop installed on OS_**

Everything works perfectly when we have DockerDesktop installed on the OS (it was fixed in [#2641](https://github.com/microsoft/vscode-docker/issues/2641))

When we do not have DockerDesktop, and we have Docker installed in Ubuntu distribution in WSL, then when we try to debug containerized Python app we get **"Timed out waiting for launcher to connect"**

At the moment the only solution I could find is to manually enter the IP bridge gateway in the **debugAdapterHost**

<details><summary>launch.json</summary>
<p>

```json
{
  "name": "Docker: Python - General",
  "type": "docker",
  "request": "launch",
  "preLaunchTask": "docker-run: debug",
  "debugAdapterHost": "172.17.0.1", // docker inspect --format '{{(index .IPAM.Config 0).Gateway}}' 'bridge'
  "python": {
    "pathMappings": [
        {
           "localRoot": "${workspaceFolder}",
           "remoteRoot": "/app"
        }
    ],
    "projectType": "general"
  }
}
```

</p>
</details>

![image](https://user-images.githubusercontent.com/15053474/168488843-f2f2aae7-623f-4c18-8325-d6c723648d0a.png)

**Comment by bwateratmsft** [comment]
I think with a minor adjustment to #3522 we can fix this.

**Comment by bwateratmsft** [comment]
This has now been released in Docker extension version 1.23.0.
